internal/sdk/ml_models.go

```diff
@@ -384,8 +384,8 @@ func (m *MlModelDisplayTagEnumValueLabel) String() string {
 type MlModelExecConfig struct {
 	ModelIdentifier string                  `json:"model_identifier"`
 	BaseUrl         string                  `json:"base_url"`
-	Metadata        map[string]interface{}  `json:"metadata,omitempty"`
-	Features        []MlModelFeature        `json:"features,omitempty"`
+	Metadata        map[string]string       `json:"metadata"`
+	Features        []MlModelFeature        `json:"features"`
```

Comment:
Note that due to a bug in the fern Go generator, I had to manually update this sdk code to remove the `,omitempty` (see [here](https://vellum-ai.slack.com/archives/C052UMCSFA8/p1723067503190829)).

The changing of `Metadata        map[string]interface{}` to `Metadata        map[string]string` was done in line with my comment above. It's just pretty hard to manage dynamic nested types in Golang/Terraform, so I decided to simplify for now.